README.md

```diff
@@ -25,3 +25,53 @@ You need to do a few things to contribute to the documentation:
 ## Styling the Docs
 
 If you want to change the look and feel of the ownCloud documentation, you can find all the information that you need in [the docs-ui repository](https://github.com/owncloud/docs-ui/blob/master/README.adoc).
+
+## How To Generate PDFs
+
+To generate a PDF version of one or more of the manuals, you need to have `asciidoctor-pdf` installed.
+With it installed, run the command below, to generate a PDF version of a manual.
+Please be aware that, depending on the size of the manual, PDF generation may take some time.
+
+```console
+asciidoctor-pdf \
+    book.adoc \
+    -a pdf-stylesdir=resources/themes \
+    -a pdf-style=owncloud
+```
+
+It invokes [asciidoctor-pdf](https://github.com/asciidoctor/asciidoctor-pdf), passing it:
+
+1. [The configuration file](https://github.com/asciidoctor/asciidoctor-pdf/blob/master/docs/theming-guide.adoc) to use. This contains the list of files to use as the PDF's source material, along with front-matter. The front-matter includes details such as whether to render a table of contents, the icon set to use, and the images base directory.
+2. The custom theme directory and the custom theme file. This ensures that the defaults are overridden, where relevant, to ensure that the generated PDF is as close to the current ownCloud style as possible.
+
+### How To Generate The PDF Configuration File
+
```

Comment:
What is the difference in ``How To Generate PDFs`` and ``How To Generate The PDF Configuration File``.
What is that necessary for ?

Reply:
The first one generates the physical (electronic) PDF file. The other generates its configuration file.

Reply:
1 is clear, but what is 2 good for and where is it needed - not clear.

From my initial reading and understanding, you would if you want to create a pdf do the configuration first and not afterwards